lib/auth/accountrecovery_test.go

```diff
@@ -1251,9 +1233,10 @@ func createUserWithSecondFactors(srv *TestTLSServer) (*userAuthCreds, error) {
 		Type:         constants.Local,
 		SecondFactor: constants.SecondFactorOn,
 		U2F: &types.U2F{
-			AppID:  "teleport",
-			Facets: []string{"teleport"},
+			AppID:  "https://localhost",
```

Comment:
Curious - why was thing change required?

Reply:
Most AppID changes are so the origin of the requests match it ("https://localhost", by default on TestDevice). An alternative was doing `WithTestDeviceOrigin("https://teleport")` all over, but this is less typing (and less code to read).

U2F is more flexible about the origin than Webauthn, hence validation isn't baked in as strongly as with Webauthn, so almost anything goes in terms of AppID for old code.